src/components/ReplayForm.vue

```diff
@@ -273,6 +286,12 @@ Civ draft: ${extractDraftUrl(civDraft.value)}`
     >
       Download
     </button>
+    <div
+      v-if="!downloadEnabled"
+      class="p-2 mt-4 text-sm text-amber-800 rounded-lg bg-ruby-100 dark:bg-ruby-400 dark:text-ruby-800"
+    >
+      {{ downloadDisabledMessage }}
+    </div>
```

Comment:
I think this is overdue: it can be hard to understand why the button is disabled.

Reply:
Yes. I like this a lot!